src/lidar/MainVis.cpp

```diff
@@ -70,8 +76,8 @@ int runRPLidar(unsigned long baudrate) {
 			double dtheta = (scan.value().angle_max-scan.value().angle_min)/(scan.value().ranges.size()-1);
 			for (long unsigned i = 0; i < scan.value().ranges.size(); i++) {
 				double rad = dtheta*i;
-				double MM_TO_M = 1000;
-				double dist = scan.value().ranges[i] * MM_TO_M;
+				double MM_PER_M = 1000;
```

Comment:
I think you've defined this `MM_PER_M` constant a few times in different `.cpp` files, so it is probably best to move it to `Constants.h` (and make it `constexpr`).